modules/nf-core/snpsites/tests/main.nf.test

```diff
@@ -0,0 +1,44 @@
+nextflow_process {
+
+    name "Test Process SNPSITES"
+    script "../main.nf"
+    process "SNPSITES"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "snpsites"
+
+    test("Test snpsites") {
+
+        when {
+            process {
+                """
+                input[0] = file(params.test_data['sarscov2']['genome']['all_sites_fas'], checkIfExists: true)
```

Comment:
```suggestion
                input[0] = file(params.modules_testdata_base_path + '/data/genomics/sarscov2/genome/alignment/all_sites.fas', checkIfExists: true)
```

Reply:
Sorry it looks, like I had a typo, but the new standard is to refer to test data like this.